src/redux/configureStore.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import countriesReducer from './reducer/countriesSlice';
+
+const store = configureStore({
+  reducer: {
+    countries: countriesReducer,
+  },
+});
+
```

Comment:
- [ ] some of your test are failing. please kindly ensure that all your test are passing.

![Screenshot (628)](https://user-images.githubusercontent.com/65298048/203873089-b9b3c5c9-693a-4ae0-851b-e343b4eb159d.png)